typings/vscode-mssql.d.ts

```diff
@@ -80,9 +80,10 @@ declare module 'vscode-mssql' {
 		 * Get the connection string for the provided connection Uri
 		 * @param connectionUri The URI of the connection to get the connection string for.
 		 * @param includePassword to include password with connection string
+		 * @param includeApplicationName to include application name with connection string
```

Comment:
`Whether the Application Name is included in the connection string. Default is true`